src/main/scala/mesosphere/marathon/storage/migration/Migration.scala

```diff
@@ -217,7 +217,8 @@ object Migration {
       StorageVersions(18, 200) -> { (migration) => new MigrationTo18200(migration.instanceRepo, migration.persistenceStore) },
       StorageVersions(19, 100) -> { (migration) => new MigrationTo19100(migration.defaultMesosRole, migration.persistenceStore) },
       StorageVersions(19, 200) -> { (migration) => new MigrationTo19200(migration.defaultMesosRole, migration.instanceRepo, migration.persistenceStore) },
-      StorageVersions(19, 300) -> { (migration) => new MigrationTo19300(migration.persistenceStore) }
+      StorageVersions(19, 300) -> { (migration) => new MigrationTo19300(migration.persistenceStore) },
+      StorageVersions(110, 0) -> { (migration) => new MigrationTo110000(migration.persistenceStore) }
```

Comment:
```suggestion
      StorageVersions(110, 100) -> { (migration) => new MigrationTo110000(migration.persistenceStore) }
```

I'd leave some room for a migration that could be slipped in before this one in case we need a fixing migration at some point.